src/pages/CartPage.js

```diff
@@ -0,0 +1,60 @@
+import { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import Card from '../components/common/Card';
+import { FaTrash } from "react-icons/fa";
+import StyledFlex from '../components/common/StyledFlex';
+import ShoppingCartContext from '../utils/state/ShoppingCartContext';
+import ItemCounter from '../components/common/ItemCounter';
+import { StyledIconWrapper } from '../components/common/StyledIconWrapper';
+const CartPage = () => {
+  const { state, dispatch } = useContext(ShoppingCartContext);
+  let history = useHistory();
+
+  if(state.items.length === 0) {
+    return <StyledFlex alignItems={'center'} justifyContent={'center'}><h1>Cart is empty</h1></StyledFlex>
```

Comment:
I don't see this text when the cart is empty.
<img width="1666" alt="Screen Shot 2022-05-17 at 16 06 42" src="https://user-images.githubusercontent.com/96157153/168910262-272d03cc-4c33-4ac1-9599-0670ea15b580.png">